.github/workflows/build.yml

```diff
@@ -51,22 +51,6 @@ jobs:
                 with:
                     node-version: "${{ matrix.node }}"
 
-            -
-                name: Require Sylius
-                run: composer require sylius/sylius:${{ matrix.sylius }} --no-interaction --no-update
```

Comment:
If this step gets deleted, then setting Sylius version in the matrix makes no sense - let's either keep it or remove Sylius version from the matrix.